src/read.jl

```diff
@@ -192,6 +194,32 @@ function read_demand_flexibility(filepath, interval)::DemandFlexibility
             )
             demand_flexibility["duration"] = interval
         end
+        
+        # Try reading the cost for down-shifting loads
+        try 
+            demand_flexibility["cost_dn"] = CSV.File(
+                joinpath(filepath, "demand_flexibility_cost_dn.csv")
+            ) |> DataFrames.DataFrame
+            println("...loading demand flexibility down-shift cost profiles")
+        catch e
+            println(
+                "Demand flexibility down-shift cost profiles not found in " * filepath
+            )
+            demand_flexibility["cost_dn"] = nothing
+        end
+
+        # Try reading the cost for up-shifting loads
+        try 
+            demand_flexibility["cost_up"] = CSV.File(
+                joinpath(filepath, "demand_flexibility_cost_up.csv")
+            ) |> DataFrames.DataFrame
+            println("...loading demand flexibility up-shift cost profiles")
+        catch e
+            println(
+                "Demand flexibility up-shift cost profiles not found in " * filepath
+            )
+            demand_flexibility["cost_up"] = nothing
+        end
```

Comment:
Can we reduce the redundancy between these two blocks? Since they both have the same structure, we can probably use a `for` loop to iterate through `["up", "dn"]`. If we want to take this approach, we can mimic what is implemented in #145 for reading the up and down demand flexibility profiles.